hackathon_site/dashboard/frontend/src/api/helpers.ts

```diff
@@ -69,6 +69,21 @@ export const teamOrderListSerialization = (
                     id: order.id,
                     status: order.status,
                     hardwareInTableRow,
+                    createdTime:
+                        "Created at: " +
+                        order.created_at.slice(0, 10) +
+                        " " +
+                        //"at " +
+                        order.created_at.slice(11, 16) +
+                        "\xa0\xa0\xa0\xa0\xa0",
```

Comment:
Some better practice for displaying text could look like this:
```javascript
const formattedDate = order.created_at.slice(0, 10);
const formattedTime = order.created_at.slice(11, 16);

const createdTime = `Created at: ${formattedDate} ${formattedTime}`;

// Consider using a CSS class or other styling techniques for spacing instead of adding multiple non-breaking spaces
```
I understand the use of \xa0\xa0\xa0\xa0\xa0 at the end of the string is to add multiple non-breaking spaces. It may achieve the desired visual effect, it's generally better to use CSS or other styling techniques for spacing.